examples/internal/proto/examplepb/visibility_rule_none_echo_service.swagger.json

```diff
@@ -112,6 +112,44 @@
           "VisibilityRuleEchoService"
         ]
       }
+    },
+    "/v1/example/echo_hidden": {
+      "get": {
+        "summary": "EchoInternalMessage method is always visible",
+        "operationId": "VisibilityRuleEchoService_EchoInternalMessage",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/examplepbHiddenMessage"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/rpcStatus"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "id",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "num",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "int64"
+          }
+        ],
```

Comment:
These parameters are inferred from the message... I guess since the fields themselves are not marked as INTERNAL this might be OK, but it still feels a bit weird.

Reply:
Was not sure if parameters should be listed when request type is hidden.